### PR TITLE
added timezone aware object

### PR DESCRIPTION
**Pull request by SmartManoj**

### CODE REVIEW

**Review by delivrance — APPROVED** [review]

**Comment by delivrance** [comment]
I've reverted this because it's causing issues when using `datetime.now() - message.date` (`TypeError: can't subtract offset-naive and offset-aware datetimes`). Maybe you should have explained beforehand why this change was needed.

**Comment by SmartManoj** [comment]
One should use `datetime.now().astimezone()`

**Review by SmartManoj — COMMENTED** [review]
[REDACTED]

File: pyrogram/utils.py [hunk 1]
```diff
@@ -53,8 +53,7 @@ def get_input_media_from_file_id(
     file_type = decoded.file_type
 
     if expected_file_type is not None and file_type != expected_file_type:
-        raise ValueError(
-            f"Expected {expected_file_type.name}, got {file_type.name} file id instead")
+        raise ValueError(f"Expected {expected_file_type.name}, got {file_type.name} file id instead")
```
Comment (SmartManoj) [inline pyrogram/utils.py:56]: What's the code width the project follows?

**Comment by delivrance** [comment]
I much prefer to allow simpler usages such as `datetime.now() + timedelta(...)` or `datetime(y, m, d, ...)` instead of having to deal with timezones. This seems to work without any issue because datetimes are only used as interface for unix timestamps, they only need to represent them.

File: pyrogram/utils.py (hunk 1 above)
Comment (delivrance) [reply pyrogram/utils.py:56]: 120 characters

**Pull request merged**